### PR TITLE
Add Support for Oasis v23

### PR DESCRIPTION
**Pull request by keefertaylor**
This PR updates dependencies and types for Oasis v23, released earlier this week.

### CODE REVIEW

**Review by VitalyV1337 — CHANGES_REQUESTED** [review]
Remove comment and LGTM

File: src/handlers/registry_handler.go [hunk 1]
```diff
@@ -689,7 +689,8 @@ func GetRuntime(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// Creating query that will be used to return runtime by it's namespace
-	query := registry.NamespaceQuery{Height: height, ID: nameSpace}
+	// query := registry.NamespaceQuery{Height: height, ID: nameSpace}
```
Comment (VitalyVolozhinov) [inline src/handlers/registry_handler.go:692]:
```suggestion
```

**Comment by keefertaylor** [comment]
Hey @VitalyVolozhinov - so sorry for not getting back to you. I missed your comment with the holidays. 

Should be good to go now.

**Comment by VitalyV1337** [comment]
Thank you @keefertaylor

**Pull request merged**